### PR TITLE
feature/members: Including al the members for stats update

### PR DESCRIPTION
**Pull request by nramirez**
image

### CODE REVIEW

File: config.js [hunk 1]
```diff
@@ -7,5 +7,5 @@ export const config = {
     PUBLIC_KEY: process.env.PUBLIC_KEY || undefined,
     LEETCODE_CHALLENGES_CHANNEL: process.env.LEETCODE_CHALLENGES_CHANNEL || undefined,
     STATS_CHANNEL: process.env.STATS_CHANNEL || undefined,
-    BOTS: (process.env.BOTS || "elprimobot").split(","),
+    BOTS: new Set((process.env.BOTS || "elprimobot").split(",")),
```
Comment (nramirez) [inline config.js:10]: BOTS.has(username) is more efficient than BOTS.includes(username)

File: api/stats.js [hunk 2]
```diff
@@ -103,21 +119,40 @@ export class Stats {
...
+            }
+
+            if (serverStats[user].posts) {
+                active.push([user, serverStats[user]]);
+            } else {
+                inactive.push(user);
+            }
+        }
+        active.sort((a, b) => b[1].posts - a[1].posts);
+        inactive.sort((a, b) => a - b);
```
Comment (nramirez) [inline api/stats.js:141]: Sorting alphabetically, however, once we start counting days of inactivity we sort by that. https://github.com/kajahno/elprimobot/issues/65

**Review by kajahno — APPROVED** [review]
LGTM, nice one! It'll be good if we pin it to an issue so there's some tracking but this is not a hard requirement (we don't have those for now)

**Comment by nramirez** [comment]
yes: this fixes https://github.com/kajahno/elprimobot/issues/57

**Pull request merged**